theme/common.less

```diff
@@ -1938,6 +1938,21 @@ div.simframe.ui.embed {
     }
 }
 
+div.signin-button {
+    display: flex;
+    align-items: center;
+
+    .text {
+        margin-right: .5rem;
+    }
+}
+
+@media only screen and (max-width: @largestTabletScreen) {
```

Comment:
This is to keep the icon floating right. The default rules are lost bc I added an extra div around the whole sign in button 

![image](https://user-images.githubusercontent.com/18712271/125887284-1f0340dc-fd01-4964-ba7f-55af5b36f335.png)
 vs 
![image](https://user-images.githubusercontent.com/18712271/125887305-29fe4d0b-4301-4957-a931-8c58017cd062.png)

Reply:
(in not desktop view)